pkg/pharos/cmd/create.go

```
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/fatih/color"
+	"github.com/lob/pharos/pkg/pharos/api"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// Declare some variables to be used as flags.
+var (
+	clusterAuthorityData string
+	server               string
+)
+
+// CreateCmd implements a CLI command that allows users to create a cluster in Pharos.
+var CreateCmd = &cobra.Command{
+	Use:     "create <cluster_id>",
+	Short:   "Creates the specified cluster",
+	Long:    "Creates the specified cluster in Pharos.",
+	Args:    func(cmd *cobra.Command, args []string) error { return argID(args) },
+	PreRunE: func(cmd *cobra.Command, args []string) error { return markFlagsRequired(cmd) },
```

is there a reason youre doing this in the `PreRun` vs just in the `init`?

---

the init doesn't allow you to return an error

---

if you follow through the source of `MarkFlagRequired`, it seems like it can only err out when the flag you're passing in isnt set on the flag set, and since we're marking them as required right after we define them, i think we can safely ignore the error

---

talked irl, we'll keep as is for now bc the linter complains if you don't handle errors